README.template.md

```diff
@@ -0,0 +1,95 @@
+# Miro Example App
+
+<b>Use this template to contribute to Miro's App Examples!</b>
+
+This app shows how to use a certain Miro SDK or API functionality to accomplish a certain task.
+
+# 👨🏻‍💻 App Demo
+
+<b>Insert app demo here.</b>
+
+# 📒 Table of Contents
+
+- [Included Features](#features)
+- [Tools and Technologies](#tools)
+- [Prerequisites](#prerequisites)
+- [Associated Developer Tutorial](#tutorial)
+- [Run the app locally](#run)
+- [Folder Structure](#folder)
+- [Contributing](#contributing)
+- [License](#license)
+
+# ⚙️ Included Features <a name="features"></a>
+
+- [Miro Web SDK](https://developers.miro.com/docs/web-sdk-reference)
+  > Insert SDK methods here which your project uses
+
+# 🛠️ Tools and Technologies <a name="tools"></a>
+
+> Insert tools and technologies here which your project uses and link to them.
+
+# ✅ Prerequisites <a name="prerequisites"></a>
+
+- You have a [Miro account](https://miro.com/signup/).
+- You're [signed in to Miro](https://miro.com/login/).
+- Your Miro account has a [Developer team](https://developers.miro.com/docs/create-a-developer-team).
+- Your development environment includes [Node.js 14.13](https://nodejs.org/en/download) or a later version.
+- All examples use `npm` as a package manager and `npx` as a package runner.
+
+> Add any other prerequisites here!
+
+# 🏃🏽‍♂️ Run the app locally <a name="run"></a>
+
+> <b>Add steps to run the app locally. Try to make them as similar as possible to other example apps for consistency.</b>
+
+1. Run `npm install` to install dependencies.
+2. Run `npm start` to start developing. \
+   Your URL should be similar to this example:
+   ```
+   http://localhost:3000
+   ```
+3. Open the [app manifest editor](https://developers.miro.com/docs/manually-create-an-app#step-2-configure-your-app-in-miro) by clicking **Edit in Manifest**. \
+   In the app manifest editor, configure the app as follows:
```

Comment:
can't we have a copy/paste example?

Reply:
I added one, lmk if it is what you were thinking? @mettin